1.0/swagger.yaml

```diff
@@ -0,0 +1,2599 @@
+swagger: '2.0'
+schemes:
+  - http
+  - https
+host: api.you2you.fr
+basePath: /api
+info:
+  description: |
+    Bienvenue sur la documentation de l'API publique You2You.
+
+    You2You est le leader de la livraison collaborative en France.
+    - Nous livrons toutes vos commandes express en 1h30 ou sur RDV dans des créneaux d’une heure.
+    - Un particulier se trouvant à proximité profite de votre demande de livraison pour rentabiliser son trajet en échange d’une compensation financière.
+    - Une livraison économique, pratique, sociale et écologique.
+
+    # Fonctionnement
+    Notre API est organisée suivant le style REST et elle :
+    - utilise des verbes et des codes de retour standards du protocole HTTP.
+    - renvoie exclusivement des réponses au format JSON ou texte.
+
+    Cette documentation est là pour vous permettre d’implémenter les mécanismes nécessaires et des exemples concrets sont présents. À ce jour, il n’existe pas de bibliothèque officielle.
+
+    # Accès à l’API
+    L’API est disponible sur deux environnements:
+
+    - L'environnement de PRODUCTION : https://api.you2you.fr
+    - L'environnement dédié aux tests : https://api-sandbox.you2you.fr
+
+    # Comment tester des requêtes vers l'API?
+    Nous recommandons l'utilisation de [POSTMAN](https://www.getpostman.com/). C'est une application pour faciliter les tests d'intégration et récupérer le code dans d'autres langages non-fournis dans notre documentation actuelle (ex: JAVA, C#, etc...).
+
+    # Versionning
+    L'API sera amenée à évoluer durant les prochains mois. Afin de ne pas perturber votre implémentation, nous veillerons impérativement à garder les points d'accès, la structure des requêtes et le format des réponses pour les versions antérieures.
+
+    Versions disponibles:
+    - [1.0](https://kangooz.github.io/apidoc/1.0)
+
+    # Récupération des clefs d'API
+    Vos clés d'API sont générées à la création de votre compte sur https://livraison.you2you.fr. Si vous êtes déjà en possession d'un compte, commencez à l'étape 2. Voici le parcours complet:
+    1. Créez votre compte You2You sur https://livraison.you2you.fr/signup.
+
+    2. Connectez-vous avec vos identifiants sur https://livraison.you2you.fr.
+
+    3. Naviguez sur votre dashboard jusqu'à la section API https://livraison.you2you.fr/api.
+
+    # Webhook
+    Un webhook est un moyen d’appeler une application tierce lorsqu’une action définie a été effectuée. Par exemple, vous pouvez déclencher l’activation d’un webhook lorsque le statut d'une livraison change ou lorsque le créneau de livraison est modifié.
+
+    Configuration de vos webhooks sur notre interface de livraison:
+
+    1. Connectez-vous avec vos identifiants sur https://livraison.you2you.fr.
+
+    2. Naviguez sur votre dashboard jusqu'à la section API https://livraison.you2you.fr/api.
```

Comment:
Can you use a target _balnk to open all link into another tab ?
ex https://github.com/Knagis/CommonMark.NET/issues/101